metadata/boundaries/municipal_boundaries.md

```diff
@@ -0,0 +1,80 @@
+
+## Brief Summary
+
+Statewide polygon dataset of municipal boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+Utah municipal boundaries defined by County Recorders' offices and certified by the Lieutenant Governor's Office. Updated boundaries from county authorities are aggregated by UGRC to produce this dataset.
+
+## Description
+
+### What is the dataset?
+
+This dataset represents the 250+ municipalities in Utah as polygons that include information such as:
+
+- Whether or not the municipality is a county seat
+- Municipality population as of the most recent Census (POPLASTCEN)
+- Estimated population of the municipality as of July 1, 2022 (POPLASTEST)
+- [GNIS](https://www.usgs.gov/faqs/what-geographic-names-information-system-gnis#:~:text=The%20Geographic%20Names%20Information%20System%20(GNIS)%2C%20developed%20by%20the,and%20outlying%20areas%20of%20the) code
+
+### What is the purpose of the dataset?
+
+Municipal boundary data are used in a variety of sectors, including city planning, real estate, and public services. In addition to displaying the extent of current municipalities, you can find a map of recent annexations and disconnections on our [blog](https://www.arcgis.com/home/webmap/viewer.html?webmap=c5ab7e0fcd514f1a9db6b8dad55bba63). Municipal data can help the public see where [annexations](https://gis.utah.gov/blog/2023-01-30-municipal-boundaries-final-update-for-2022/) have occurred in the past and where [new cities and towns](https://gis.utah.gov/blog/2018-09-04-newest-city-added-to-state-and-municipal-boundary-updates-for-third-quarter-2018/) have been established in Utah.
+
+### What does the dataset represent?
+
+This dataset does not represent exact legal boundaries, but, rather a set of polygon boundaries used for general purposes that conforms to logical and administrative rules (e.g. no two cities or redevelopment areas may cover the same geographic extent).
+
+### How was the dataset created?
+
+This dataset was first created through a cooperative effort between the [Governor's Office of Planning and Budget](https://gopb.utah.gov/), [UDOT](https://udot.utah.gov/), the [Utah State Tax Commission](https://incometax.utah.gov/), and UGRC. Polygon features for each municipality were created using coordinate geometry [(COGO)](https://pro.arcgis.com/en/pro-app/latest/help/editing/introduction-to-cogo.htm) from legal descriptions. All features have been adjusted to the current Geographic Coordinate Database (GCDB) points.
+
+The original dataset incorporated municipalities in the state that were extant as of 1992. Updates for further annexations and disconnections have been completed with the cooperative efforts of the State Tax Commission, [Property Tax Division](https://propertytax.utah.gov/), and UGRC. Updates to the municipal boundaries proposed by counties or other entities must be certified through the [Lieutenant Governor's Office](https://ltgovernor.utah.gov/).
```

Comment:
```suggestion
The original dataset contained municipalities that were extant as of 1992. Since then, UGRC has updated it to reflect additional annexations and disconnections certified through the [Lieutenant Governor's Office](https://ltgovernor.utah.gov/) in cooperation with the State Tax Commission [Property Tax Division](https://propertytax.utah.gov/).
```

Reply:
Adjusted the wording a bit, let me know what you think!

Reply:
So much better than my convoluted sentence. I love it.